app/apps/ood_app_group.rb

```diff
@@ -34,7 +34,17 @@ def self.select(titles:[], groups:[])
     # only one group per title; this just makes it easy to get the group
     # Hash[ [:title1,:group1], [:title2,:group2]] => {title1: :group1, title2: :group2 }
     h = Hash[groups.map {|g| [g.title, g]}]
-
     titles.map { |t| h.has_key?(t) ? h[t] : nil }.compact
   end
+  
+  # select a subset of groups by the specified array of titles
+  # maintaining the order specified by the titles array
```

Comment:
Fix the comment here since we aren't filtering out any groups, so "select a subset of groups by the specified array of titles" is not true.